euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+
+int gcd_rec(int a, int b)
+{
+    return b == 0 ? a : gcd_rec(b, a % b);
```

Comment:
Line contains following spacing inconsistencies:
- Spaces used instead of tabs.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwazjr6gq/euclidean_gcd/CPP/euclidean_gcd.cpp
+++ b/tmp/tmpwazjr6gq/euclidean_gcd/CPP/euclidean_gcd.cpp
@@ -2,7 +2,7 @@
 
 int gcd_rec(int a, int b)
 {
-    return b == 0 ? a : gcd_rec(b, a % b);
+	return b == 0 ? a : gcd_rec(b, a % b);
 }
 
 
```